tests/test_route_scope.py

```diff
@@ -47,4 +47,4 @@ def test_websocket():
 def test_websocket_invalid_path_doesnt_match():
     with pytest.raises(WebSocketDisconnect):
         with client.websocket_connect("/itemsx/portal-gun"):
-            pass
+            pass  # pragma: nocover
```

Comment:
For some reason, locally I need this to have 100% coverage
I can revert this to keep the PR cleaner if needed